cpp/test/CMakeLists.txt

```diff
@@ -185,6 +185,7 @@ if(BUILD_PRIMS_TESTS)
     prims/multiply.cu
     prims/mutualInfoScore.cu
     prims/norm.cu
+    prims/nvtx_test.cu
```

Comment:
This file should be part of target `ml` and not part of `prims`. Currently `nvToolsExt` is not part of `prims` list of linked libraries, should it be added?

Reply:
Ouch.. that's a good catch Vinay! @venkywonka can you file another PR for this, please?